profiles/local-frontend.js

```diff
@@ -10,4 +12,12 @@ routes[`/${SECTION}/${APP_ID}`]      = { host: `https://localhost:${FRONTEND_POR
 routes[`/beta/apps/${APP_ID}`]       = { host: `https://localhost:${FRONTEND_PORT}` };
 routes[`/apps/${APP_ID}`]            = { host: `https://localhost:${FRONTEND_PORT}` };
 
-module.exports = { routes };
+const esi = {
+    // Increases the default (2s) timeout which can be a pain sometimes.
+    // https://github.com/Schibsted-Tech-Polska/good-guy-http/blob/master/lib/index.js#L55
+    httpClient: goodGuyLib({
+        timeout: 5000
```

Comment:
Can we set it to 5minutes i.e. 600 seconds? Since the default is 2s, 600s should help us with sufficient time in hand, coz more than 40minutes seems like a big waiting time.
What do you think?

Reply:
I don't mind modifying it to `600` sec. I have just copied config from https://github.com/RedHatInsights/compliance-frontend/blob/master/config/spandx.config.js#L58 from slack discussion.

Reply:
I will make the change.  From slack discussion, I got the reference - https://github.com/RedHatInsights/compliance-frontend/blob/master/config/spandx.config.js#L58 so I just copied it to our config.

Reply:
From slack discussion, I got the reference - https://github.com/RedHatInsights/compliance-frontend/blob/master/config/spandx.config.js#L58 so I just copied it to our config.
Sorry, I didn't notice previously. Default value is 2s i.e 2000 ms, currently it sets to 5000ms i.e 5s 
Do you want me to set 5 mins i.e 300sec (300000 ms) right? 
Where it says 40 minutes or given as an example?

Reply:
If any changes required in timeout value, I will update it separately.  For now merging it. 
Thanks @apuntamb!